controllers/articles.controller.js

```diff
@@ -87,9 +87,29 @@ function deleteComByComment_id(request, response, next){
         }
       })
     .catch((error)=>{
-        console.log(error)
         next(error)
     })
 }
 
-module.exports = { getArticleById, getComByArticle_id, postComByArticle_id, patchVotesByArticle_id, deleteComByComment_id }
+function getAllArticles (request, response, next){
+   const { topic } = request.query
+    selectAllArticles(topic)
+    .then((result)=>{
+        if (result.rows.length === 0){ //promise reject where articles doesn't exists
```

Comment:
if there are no articles for this topic, that's not necessarily an error - there just any articles in the same way an article that has no comments, we send an empty array rather than an error to say no comments. An error may interfere with our ability to load the page/the site